templates/kubernetes/docs/certs-and-trust.md

```diff
@@ -0,0 +1,156 @@
+---
+wrapper_template: "base_docs.html"
+markdown_includes:
+  nav: "shared/_side-navigation.md"
+context:
+  title: "Certificates and Trust Management"
+  description: How CDK manages PKI certificates and trust
+keywords: juju, kubernetes, cdk, security, pki, encryption, vault, ha
+tags: [operating, security]
+sidebar: k8smain-sidebar
+permalink: certs-and-trust.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+The components of **Canonical Distribution of Kubernetes<sup>&reg;</sup> (CDK)**
```

Comment:
it is now called 'Charmed Distribution of Kuberneted' and CDK and Charmed Kubernetes for short

Reply:
Okay, I think I got them all